tests/benchmarks/MicroPerf/Collections.fs

```diff
@@ -0,0 +1,298 @@
+module Collections
+
+open BenchmarkDotNet.Attributes
+open BenchmarkDotNet.Running
+open BenchmarkDotNet.Configs
+open BenchmarkDotNet.Diagnosers
+open System.Runtime.CompilerServices
+
+[<SimpleJob(launchCount = 2, warmupCount = 1, targetCount = 2)>]
+[<GcServer(true)>]
+[<MemoryDiagnoser>]
+[<MarkdownExporterAttribute.GitHub>]
+type CollectionsBenchmark() =
+
+    [<Params( (* 0, 1, 100, *) 1000, 10000)>]
+    member val Length = 0 with get, set
+    /// List
+    [<Benchmark>]
+    member x.ListRemoveAtBeginning() =
+        async {
+            List.init x.Length (fun _ -> 0)
+            |> List.removeAt 0 
+            |> ignore
+        }
+        |> Async.StartAsTask
+
+    [<Benchmark>]
+    member x.ListRemoveAtEnd() =
+        async {
```

Comment:
Is the use of tasks to run the benchmarks in parallel?  It's a bit odd as there will be allocation overhad from async and StartAsTask

Reply:
Honestly, I just copied what the other benchmarks were doing without giving it much thought. I removed the async/tasks now.

Reply:
Yup I hadn't reviewed those - thanks!